seed/views/v3/derived_columns.py

```diff
@@ -172,3 +176,75 @@ def destroy(self, request, pk):
             'status': 'success',
             'message': 'Successfully deleted derived column',
         })
+
+    @swagger_auto_schema(
+        manual_parameters=[
+            AutoSchemaHelper.query_org_id_field(),
+            AutoSchemaHelper.query_integer_field(
+                name='cycle_id',
+                required=True,
+                description='Cycle to evaluate'
+            ),
+            openapi.Parameter(
+                'inventory_ids',
+                openapi.IN_QUERY,
+                description='List of inventory IDs (i.e. Property or TaxLot)',
+                required=True,
+                type=openapi.TYPE_ARRAY,
+                items=openapi.Items(
+                    type=openapi.TYPE_INTEGER
+                )
+            )
```

Comment:
I don't want to expand scope and context switch too much, but as a "quickfix", what do you think about leaving a code comment here to revisit this and standardizing this via AutoSchemaHelper?

Reply:
I'm not too worried about this yet, this is the only place so far we've had to create this custom swagger parameter, maybe once it pops up some more we can push it into the AutoSchemaHelper